### PR TITLE
BOM-1012

### PR DESCRIPTION
**Pull request by awais786**
Add tox environments for Django 2.0, 2.1, and 2.2 under Python 3.5
https://openedx.atlassian.net/browse/BOM-1012

### CODE REVIEW

**Comment by awais786** [comment]
@jmbowman Please review.

File: tox.ini [hunk 1]
```diff
@@ -16,6 +16,9 @@ norecursedirs = .* docs requirements
 [testenv]
 deps =
     django111: Django>=1.11,<2.0
+    django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
     -r{toxinidir}/requirements/test.txt
 commands =
     py.test {posargs}
```
Comment (jmbowman) [inline tox.ini]: I think that in order to display deprecation warnings, we need to update this to something like `python -Wd -m pytest {posargs}` (although there might not be any warnings triggered by the code in this repo, it's pretty small).

File: requirements/dev.txt [hunk 2]
```diff
@@ -18,7 +18,7 @@ contextlib2==0.6.0.post1  # via importlib-metadata
 diff-cover==2.4.0
 distlib==0.3.0            # via caniusepython3
 django-config-models==1.0.1
-django==1.11.25
+django==1.11.26 ; python_version < "3"
```
Comment (jmbowman) [inline requirements/dev.txt]: Hmm, this isn't really what we want in the output; Django won't be correctly pinned in Python 3 dev environments.  We may need to give up on the upper constraint on Django in `base.in` and just leave it as `Django>=1.11`; I don't think there's a good way to get `pip-compile` to generate what we want here with an environment marker involved (there's an [open issue](https://github.com/jazzband/pip-tools/issues/518) for this).

**Comment by awais786** [comment]
@jmbowman PR is ready for final review.

**Pull request merged**